constants/fallFest2023Content.ts

```diff
@@ -0,0 +1,95 @@
+import { DescriptionCard } from "~/types/uiComponents";
+
+const header = {
+  titleLine1: "Qiskit Fall Fest 2023",
+  description:
+    "The Qiskit Fall Fest is a collection of quantum computing events on college campuses around the world. Every Fall Fest event is organized and run by the students on each campus, with support from IBM Quantum. Student leaders work alongside IBM to design and plan their event, which are run in October and November. Each event is standalone - only made for the students on that particular campus. The events vary from technical hackathons to beginner-friendly challenges, all with the goal of helping to grow the local community’s skill set.",
+  cta: {
+    label: "2022 Fall Fest events",
+    url: "/events/fall-fest-2022",
+    segment: {
+      cta: "learn",
```

Comment:
I don't think `learn` is a good cta, we need to replace it with a better one

Reply:
addressed in https://github.com/Qiskit/qiskit.org/pull/3530/commits/49621699296972dc357619bb777e36459495df76